django/db/models/sql/constants.py

```diff
@@ -35,3 +35,8 @@
 # SQL join types.
 INNER = 'INNER JOIN'
 LOUTER = 'LEFT OUTER JOIN'
+
+# Set operators
+UNION = 'union'
+INTERSECT = 'intersect'
+EXCEPT = 'except'
```

Comment:
@felixxm Can you provide some input on what and how oracle support would look like?